src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawing {
```

Comment:
```suggestion
public class Circle extends Figure {
```
if parent class implements an interface, all child classes implement it as well